scavenger-api/src/main/kotlin/com/navercorp/scavenger/dto/ExportMethodInvokationDto.kt

```diff
@@ -0,0 +1,29 @@
+package com.navercorp.scavenger.dto
+
+import com.navercorp.scavenger.entity.ExportMethodInvocationEntity
+
+data class ExportMethodInvokationDto(
```

Comment:
```suggestion
data class ExportMethodInvocationDto(
```
Small typo here!

Reply:
Thank you! I renamed all the components from ExportMethodInvocation to ExportSnapshotMethod.